tests/foreman/cli/test_rhcloud_inventory.py

```diff
@@ -139,3 +132,79 @@ def test_positive_sync_inventory_status():
 
     :CaseLevel: System
     """
+
+
+@pytest.mark.stubbed
+def test_rhcloud_inventory_bz_1962694():
```

Comment:
I don't think naming the test case for the BZ is a good practice. The BZ context is available in the metadata, and provides zero context when viewing test results concerning what the test function is for. Can't tell a simple field validation test from a complex integration test.

Please name the test cases according to their relevant coverage. 

Is there no way to automate any of these three test cases, as indicated by `ManualOnly` metadata?

Reply:
@mshriver updated the pr.